### PR TITLE
Adding a template to create a Linux VM with new version of Azure Linux Agent from Github.

### PR DESCRIPTION
**Pull request by SuperScottz**
Updated the metadata and README per comment.

### CODE REVIEW

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+        },
+        "dnsNameForPublicIP" : {
+            "type" : "string",
+            "metadata": {
+            "Description": "the new DNS name."
+             }
+        },
+        "adminUserName": {
+            "type": "string"
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: missing comma

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+            "metadata": {
+            "Description": "the admin user name."
+             }
+        },
+        "adminPassword": {
+            "type": "securestring"
+            "metadata": {
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: missing comma

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            "Description": "the password for admin user."
+             }
+        },
+        "vmSourceImageName": {
+            "type": "string",
+            "defaultValue": "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-14_04_2_LTS-amd64-server-20150309-en-us-30GB"
+            "metadata": {
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: missing comma

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                "type": "CustomScriptForLinux",
+                "typeHandlerVersion": "1.2",
+                "settings": {
+                    "fileUris": [
+                        "https://raw.githubusercontent.com/azurermtemplates/azure-quickstart-templates/master/vm-linuxagent-upgrade/vm-linuxagent-upgrade.sh"
+                    ],
+					 "commandToExecute": "[concat('sh vm-linuxagent-upgrade.sh ', parameters('LinuxAgentVersion'))]"
+                }
+            }
+        }
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: Github is showing red in this entire file which means it's missing commas and quotes. Please run this through a JSON linter to discover errors

File: vm-linuxagent-upgrade/azuredeploy-parameters.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+﻿{
+    "newStorageAccountName" : {
+        "value":"uniqueStorageAccountName"
+    },
+    "adminUsername": {
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy-parameters.json]: Please supply dummy values for the parameters

File: vm-linuxagent-upgrade/metadata.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Linux VM with new version of Azure Linux Agent from Github using custom script Linux extension",
```
Comment (singhkay) [inline vm-linuxagent-upgrade/metadata.json]: Name too long. Suggestion: Install Azure Linux Agent on a VM

**Comment by singhkay** [comment]
Any update on this? @SuperScottz

**Comment by SuperScottz** [comment]
@singhkay I'm going to create an new pull requrest with some correction based on comment soon.

**Comment by SuperScottz** [comment]
See new pull request: https://github.com/Azure/azure-quickstart-templates/pull/294
please review.

**Pull request closed without merging**